build/test-scripts/integration/run-kubernetes-tests-on-vm-agents.sh

```diff
@@ -13,7 +13,7 @@ $SCRIPT_DIR/common/setup-vm-agent.sh
 
 source $SCRIPT_DIR/common/set-dotnet-envvars.sh
 
-itdll=`pwd`/build/outputs/integrationtests/net6.0/linux-x64/Octopus.Tentacle.Kubernetes.Tests.Integration.dll
+itdll=`pwd`/build/outputs/integrationtests/net8.0/linux-x64/Octopus.Tentacle.Kubernetes.Tests.Integration.dll
```

Comment:
❓ This might be worth changing to use an environment variable (like `run-integration-tests-on-vm-agents.sh`)